Gopkg.toml

```diff
@@ -26,8 +26,8 @@
 
 
 [[constraint]]
-  branch = "ss-master"
-  name = "github.com/shipstation/go-teamcity-sdk"
+  name = "github.com/cvbarros/go-teamcity-sdk"
+  source = "../go-teamcity-sdk"
```

Comment:
The `source` has to be relative to your $GOPATH, it cannot be anywhere you want.

Reply:
Scratch that, it is the repo on github..., nothing to do with the file system.